docs/reference/commandline/volume_ls.md

```diff
@@ -168,10 +168,10 @@ Valid placeholders for the Go template are listed below:
 
 Placeholder   | Description
 --------------|------------------------------------------------------------------------------------------
-`.Name`       | Network name
-`.Driver`     | Network driver
-`.Scope`      | Network scope (local, global)
-`.Mountpoint` | Whether the network is internal or not.
+`.Name`       | Volume name
+`.Driver`     | Volume driver
+`.Scope`      | Volume scope (local, global)
+`.Mountpoint` | Whether the volume is internal or not.
 `.Labels`     | All labels assigned to the volume.
 `.Label`      | Value of a specific label for this volume. For example `{{.Label "project.version"}}`
```

Comment:
It does have `.Label` still;

```bash
$ docker volume create --label project.version=one-two foobar
$ docker volume create --label project.version=three-four foobar2

$ docker volume ls --format 'table {{.Name}}\t{{.Label "project.version"}}'

VOLUME NAME         VERSION
foobar              one-two
foobar2             three-four
```

w.r.t. the `Links` and `Size` - these values are only propagated by the "prune" command; I'm actually looking at deprecating those fields for `volume ls`, and using a dedicated API type for the prune action